README.md

```diff
@@ -9,11 +9,14 @@
 
 Daemons for [Andor](https://andor.oxinst.com/) cameras
 
+Sdk3 files
```

Comment:
I think it's important to show how daemons need different drivers.  But the package contains list should have all daemons regardless of SDK used.  We can have a separate section describing differences in installation details/drivers needed.

Reply:
This may be resolved.